### DISCUSSION TITLE
MIB AUDI A5 F5 B9

### DISCUSSION THREAD

**Discussion by PierosA5** [Mr-MIBonk/M.I.B._More-Incredible-Bash#21] [Q&A] [answered]
Hi, I apologize for my English but I am Italian.
I read some forums and came to this site.
I have a 2019 AUDI A5 F5 B9 and I would like to use your patches (with SD Card) on my MIB but I have doubts about what to do.
Up to now I have only made encodings with VCDS and unlocked the VIM with OBD eleven.
The FEC licenses for all CONNECT services have expired on my MIB.
The Maxar satellite maps (Google Earth) have also expired.
I ran out of six free navigation map updates.
My MIB is not equipped with an AUDI SMARTPHONE INTERFACE that I would like to activate and mount as well as the USB data (I have the USB only charged)
Up to now I have not activated the GEM.
The VERSION of my MIB is: MHI2Q_ER_AUG22_P5092
Software: MU 1329.

Looking at your site I saw several files and folders, I got stuck for fear of not getting patches wrong.

Can you help me find the patch to unlock the features I've listed?
My car is still under Audi warranty, are there any errors in the diagnosis with ODIS / VCDS?
If I follow your instructions carefully I can risk blocking my MIB forever.

**Comment by harman-f** [comment]
I assume, that you have not seen the Wiki article yet?
Please read:
https://github.com/Mr-MIBonk/M.I.B._More-Incredible-Bash/wiki/Prepare-M.I.B-SD-card-&-install-M.I.B#i---prepare-mib-sd-card
Patch will be automatically selected fitting for your FW.
Just copy them all to SD.

Patch will not activate Connect, you still have to buy yearly license from Audi.
Satellite view is also linked to valid connect license.

MJ2019, still in warranty, but already used 6 Nav updates? This does not fit.
Also charge only USB would be strange for a MJ2019 car. Please post a picture of your USB port.

As long as you do not power down your unit during flashing nothing bad can happen.

**Reply by PierosA5** [reply]
So I didn't understand, what do I get with my MIB patch?
You can explain it to me in a simple way.
I attach photos of the USB (in Italy the USB data is a non-standard optional for the 2019 Audi)
Thanks friend

**Reply by harman-f** [reply]
You will be able to update maps unlimited.

I can not see pictures attached of your USB port.

After adding new USB port also CarPlay and AndroidAuto can work.

**Reply by PierosA5** [reply]
Ok I understand now, free map updates and of course GEM unlock.
But will I be able to download the maps directly to my MMI (as before the free updates ran out) with the internet connection of the car?
Or will I have to download them to PC and then install with SD?
Thanks friend
I try to attach the Dell USB photo again.
Can you confirm that in the diagnosis with VCDS or OBD Eleven there will be no error?
![IMG_20220724_172920](https://user-images.githubusercontent.com/108060323/180661839-255b452b-f2a1-4f48-9d3f-9c1f7ec81542.jpg)
![IMG_20220723_115824](https://user-images.githubusercontent.com/108060323/180661843-bbc68f5a-5489-4647-aae0-d643a5babd79.jpg)

**Comment by harman-f** [comment]
Maps only via SD card.

Seems you need to get new USB port for CP/AA

There will be no errors

**Reply by PierosA5** [reply]
I'll take a new USB port and the Montero. Where do I download the maps for my car?

**Reply by harman-f** [reply]
https://navigation-maps.volkswagen.com/vw-maps/P197_N60S5MIBH3_EU.7z

**Reply by harman-f** [reply]
and yes, maps are all the same for the brands, using the once from VW is OK

**Reply by PierosA5** [reply]
Do I have to leave the SD inserted in the MMI to make the patch work? If I remove it, does it go back to the original firmware?

**Comment by harman-f** [comment, marked as answer]
AFTER all is installed you can remove SD card and patch will stay.

**Reply by PierosA5** [reply]
I would like to understand, if it is not disturbing, why if you can generate FEC for the maps for life and for the Smartphone Interface it is not possible to patch the Audi Connect services including Google Heart?
Will we always be forced to pay € 260 every year or are we working on it?

**Reply by harman-f** [reply]
Different topic. Connect is not FEC related, but custom built server is required.
